docs/software/file_servers.md

```diff
@@ -120,6 +120,56 @@ The Samba server lets you share files on your DietPi system with ease based on t
     systemctl restart nmbd smbd
     ```
 
+=== "Use `wsdd` to view shares"
+
+    In case of problems with the Samba share not showing up in the Windows network view, the `wsdd` daemon (Web Service Dynamic Discovery host daemon) can be installed. This can be done with the following steps:
+
+    1. Install `wsdd`:
+       
+        ```sh
+        wget -O- https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key | gpg --dearmour > /usr/share/keyrings/dietpi-wsdd.gpg
```

Comment:
Let's stick with our convention, so once we implement in `dietpi-software`, users who followed this guide won't end up with doubled keys and such:
```suggestion
        curl -sSfL 'https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key' | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-wsdd.gpg --yes
```